src/huggingface_hub/serialization/_tensorflow.py

```diff
@@ -27,7 +28,7 @@
 def split_tf_state_dict_into_shards(
     state_dict: Dict[str, "tf.Tensor"],
     *,
-    filename_pattern: str = "tf_model{suffix}.h5",
+    filename_pattern: str = constants.TF2_WEIGHTS_FILE_PATTERN,
```

Comment:
I'm not sure if it falls into the scope of this PR, but it would be nice to include some notes on how to adjust the `filename_pattern` when it's non-default. I remember having to go through a couple hoops to get it right for `diffusers` filename pattern.

Reply:
Disregard.